src/implicit_function.jl

```diff
@@ -61,8 +52,8 @@ function ChainRulesCore.frule(
     pushforward_A(dỹ) = frule_via_ad(rc, (NoTangent(), dỹ), conditions_y, y)[2]
     pushforward_B(dx̃) = frule_via_ad(rc, (NoTangent(), dx̃), conditions_x, x)[2]
 
-    mul_A!(res, u::AbstractVector) = res .= vec(pushforward_A(reshape(u, size(y))))
-    mul_B!(res, v::AbstractVector) = res .= vec(pushforward_B(reshape(v, size(x))))
+    mul_A!(res::Vector, u::Vector) = res .= vec(pushforward_A(reshape(u, size(y))))
```

Comment:
JET on Julia v1.8 threw an error there because of broadcasting. For lack of a typed `res` argument, the operation `res .= ...` was potentially ill-defined. Since we convert the input to `Vector` anyway, I figured there was no loss involved.

That's also why I added tests on v1.8

Reply:
fair enough